GitCommands/RevisionGraph.cs

```diff
@@ -22,7 +22,7 @@ public enum RefsFiltringOptions
         ShowGitNotes = 32,          // --not --glob=notes --not
         NoMerges = 64,              // --no-merges
         FirstParent = 128,          // --first-parent
-        SimplifyByDecoration = 256  // --simplify-by-decoration
+        SimplifyByDecoration = 256 // --simplify-by-decoration
```

Comment:
:bug: This looks like a bug. Add a comma after `256` and the problem should be resolved with the comments still aligned.